x-pack/plugins/observability_solution/infra/public/components/ml/anomaly_detection/anomalies_table/anomalies_table.tsx

```diff
@@ -135,12 +176,31 @@ const AnomalyActionMenu = ({
 
   if (!disableShowInInventory) {
     items.push(
-      <EuiContextMenuItem key="showInInventory" icon="search" onClick={showInInventory}>
-        <FormattedMessage
-          id="xpack.infra.ml.anomalyFlyout.actions.showInInventory"
-          defaultMessage="Show in Inventory"
-        />
-      </EuiContextMenuItem>
+      influencerField === HOST_NAME_FIELD ? (
+        <EuiContextMenuItem
+          key="showInHostsList"
+          icon="search"
+          data-test-subj="infraAnomalyFlyoutShowInHostsList"
+          onClick={showInHostsList}
+        >
+          <FormattedMessage
+            id="xpack.infra.ml.anomalyFlyout.actions.showInHostsList"
+            defaultMessage="Show in Hosts List"
```

Comment:
@roshan-elastic I'm slightly confused about "Hosts List". This doesn't match with the side nav, I wonder if `Show in Hosts` makes more sense.

Reply:
Yeah sorry, 'Show in Hosts' makes more sense. Cheers @crespocarlos